1.0/docs/devguide/styling.md

```diff
@@ -24,92 +40,148 @@ provided via `<style>` tags placed inside the element's local DOM `<template>`.
+required class, `my-element`, to the created element (and to the selector).
+
+
+### Styling distributed children (`::content`)
+
+In shadow DOM, you can style an element's light DOM using the `::content` pseudo element. So, you can do this:
+
+    <dom-module id="my-element">
+
+      <template>
+
+        <style>
+          /* WARNING: This does NOT work in shady DOM */
+          ::content > .special {
+            background: orange;
+          }
+        </style>
 
-### Styling distributed children (::content)
+       <div class="special">This should not be affected</div>
+       <content></content>
+      </template>
 
-Under shady DOM, the `<content>` tag doesn't appear in the DOM tree. Styles are rewritten to remove the
-`::content` pseudo-element, **and any combinator immediately to the left of `::content`.**
+      <script>
+          Polymer({
+              is: 'my-element'
+          });
+      </script>
 
-This implies:
+    </dom-module>
 
-*   You must have a selector to the left of the `::content` pseudo-element.
 
-        :host ::content div
+In shadow DOM, the `.special` class will only ever affect elements in the light
+DOM. So when you use your element:
 
-    Becomes:
+    <my-element>
+        <div class="special">This should be special!</div>
+    </my-element>
 
-        x-foo div
+Only `This should be special!` will have an orange background.
 
-    (Where `x-foo` is the name of the custom element.)
+#### Shim limitations
 
-*   To limit styles to elements inside the ::content tag, add a wrapper element around the
-    `<content>` element. This is especially important when using a child combinator (`>`) to
-    select top-level children.
+Shady DOM is much more limited than shadow DOM.
 
-        <dom-module id="my-element">
+Under shady DOM,  the `<content>` tag doesn't appear in the DOM tree, and styles
+are rewritten to remove the `::content` pseudo-element **as well as and any
+combinator immediately to the left of `::content`.** (E.g. `> :: content` where `>`
+is the combinator before it).
 
-          <template>
+Using shady DOM, the example above sees the style `::content > .special {`
+is rewritten as `my-element > .special`. This means that the rule will not apply
+just to the light DOM (as it should), but to the local DOM too (which it
+shouldn't).
 
-            <style>
-              .content-wrapper ::content > .special {
-                background: orange;
-              }
-            </style>
+**NOTE: The original documentation is very confusing here, as it says that you must have a selector before `::content` (for example `:host`), but in my experience that doesn't actually do anything since BOTH `:host ::content div` and `::content div` becomes `element div` (which is obviously not enough). I wonder if adding `element` was a recent-ish addition, and earlier versions of polymer had `::content div` turn into a dangerous, leaky `div`!**
```

Comment:
Yeah, I think  the doc is confusing here. Several things got conflated. 

At one point, you just needed a selector there.

But if you're using a child combinator, or if you have anything else in the local DOM that could be affected by these styles, you should use the wrapper approach. 

You highlighted one of the possible issues, here's the other one:

Rewriting `::content > .special` as `my-element > special` is potentially wrong because `>` selects _direct children`. So consider this element template:

    <template>
      <div id="main">
        <content></content>
      </div>
    </template>

Used like this:

    <my-element>
      <div class="special">I'm special!</div>
      <div>I'm not.</div>
    </my-element>
      
If I use the rule `::content > .special`, and in shady DOM that gets transformed to `my-element > .special` ... And it no longer selects anything, because the _rendered_ tree looks like this:

    <my-element>
       <div id="main">
         <div class="special">I'm special!</div>
         <div>I'm not.</div>
       </div>
    </my-element>

So, short version: If you have a very simple local DOM template like this:

    <template>
      <content></content>
    </template>

You can get away with just `::content selector`.

If your element has other local DOM contents that could match `selector`, or if you want to select only the top-level distributed children (using the child combinator (`>`)), then you need to use a wrapper.